backend/configs/main.conf

```diff
@@ -17,4 +17,6 @@ camera_4_address = 192.168.10.101
 serverip=localhost
 serverport=8888
 sqluser=root
-sqlpassword=root
\ No newline at end of file
+sqlpassword=root
```

Comment:
Maybe we should pass these as command line parameters when starting the servers (maybe keep these as default if they're not provided in the command line).